### PR TITLE
Config mvh

### PR DESCRIPTION
**Pull request by hasentopf**
Little House Keeping PR

### CODE REVIEW

File: app/configurations/config.herrenberg.js [hunk 1]
```diff
@@ -397,8 +396,6 @@ export default configMerger(parentConfig, {
         'lot_type:Parkhaus'
     ],
 
-    realtime: { hbg: realtimeHbg },
-
```
Comment (hbruch) [inline app/configurations/config.herrenberg.js:401]: Are you sure, that this should be removed?

**Review by hbruch — CHANGES_REQUESTED** [review]
I think that the realtime configuration should not be removed. Why do you think so?

File: app/configurations/config.herrenberg.js (hunk 1 above)
Comment (hasentopf) [reply app/configurations/config.herrenberg.js:401]: See Line: 238

Comment (andreashelms) [reply app/configurations/config.herrenberg.js:401]: Yes, looks like this was duplicated. `realtime` config is still there.

**Review by andreashelms — APPROVED** [review]

**Review by hbruch — APPROVED** [review]

**Pull request merged**